src/Service/Database/ListTablesService.php

```diff
@@ -35,14 +35,15 @@ public function run(): int
     {
         $tables = $this->getAllTableNames();
         $migrationTable = $this->db->getSchema()->getRawTableName($this->migrationService->getMigrationTable());
+        $dsn = $this->db->getDSN();
 
-        if (empty($tables) || implode(',', $tables) === $migrationTable) {
+        if (empty($tables) || implode(',', $tables) === $migrationTable || $dsn === null) {
```

Comment:
I'd use separate error for empty DSN.

Reply:
If I knew you would say it :), 

Fixed.